docs/releasenotes/2.8.0.rst

```diff
@@ -0,0 +1,53 @@
+:orphan:
```

Comment:
We should also update `docs/cli_options.rst:52-55` file accordingly and the description of this CLI command in `robocop/config.py:229`.